gpmp2/kinematics/Arm.h

```diff
@@ -37,68 +37,85 @@ class GPMP2_EXPORT Arm
   std::vector<gtsam::Pose3>
       link_trans_notheta_;  // transformation of each link, no theta matrix
 
+  bool modDH_;  /// Boolean to switch to modified Denavit-Hartenberg
+                /// parameterization
```

Comment:
If a docstring is not a one-line comment, it should go above the statement. That way doxygen is able to pick it up correctly.